src/routes/transactions/mappers/common/custom-transaction.mapper.spec.ts

```diff
@@ -192,7 +192,7 @@ describe('Multisig Custom Transaction mapper (Unit)', () => {
     const dataSize = 0;
     const chainId = faker.string.numeric();
     const transaction = multisigTransactionBuilder().build();
-    const humanDescription = faker.word.words();
+    const humanDescription = [];
```

Comment:
We should adjust the test to generate human descriptions that are ready to be consumed by the clients.

We can leverage the builder pattern to make it easier to write those tests 😉

Reply:
Good idea, I've created a new builder for human description fragments and used it in this test.